packages/web/src/components/spa/HackAUBG/MentorsSection/MentorsSection.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import axios from 'axios';
+import { useState } from 'react';
+import { useEffect } from 'react';
+import { url, openNewTab } from '../../../../Global';
+import { MentorsCard } from './MentorsCard';
+import './mentors_section.css';
+
+export const MentorsSection = () => {
+    const [mentor, setMentors] = useState([{}]);
+
+    const getMentors = () => {
+        axios({
+            method: 'get',
+            url: url + '/api/mentors'
+        })
+            .then((res) => {
+                setMentors(res.data.data.data);
+            })
+            .catch((err) => {
+                console.log(err);
+            });
+    };
+
+    const renderMentors = () => {
+        if (mentor) {
+            return (
+                <div className="mentors-container">
+                    <h1 className="mentors-header">Mentors</h1>
+                    <div className="mentors-picture">
+                        {mentor.map((mentor, index) => (
+                            <div key={index} className="mentors-div">
+                                <MentorsCard
+                                    Mentor={mentor}
+                                    openNewTab={openNewTab(mentor.sociallink)}
```

Comment:
rework logic so that you only pass the url. 

`onClick={()=>{openNewTab(url)}}`

Scratch that. 
You don't need to pass anything else except the mentor prop. 

Only thing to change is the onClick -> 

`onClick={()=>{openNewTab(mentor.sociallink)}}`

Reply:
Doesn't work because the lambda executes openNewTab function for each mentor when it's loaded, not when it's clicked.

Reply:
I pushed the changes for the Mentors section, review them and reapply for Judges